qupath-gui-fx/src/main/java/qupath/lib/gui/commands/Commands.java

```diff
@@ -1724,10 +1724,19 @@ public static void convertDetectionsToPoints(ImageData<?> imageData, boolean pre
 		boolean	deleteDetections = button == ButtonType.YES;
 		PathObjectTools.convertToPoints(hierarchy, pathObjects, preferNucleus, deleteDetections);
 		imageData.getHistoryWorkflow().addStep(
-				new DefaultScriptableWorkflowStep("Convert detections to points",
-						String.format("PathObjectTools.convertToPoints(getCurrentHierarchy(), getDetectionObjects(), %b, %b)", preferNucleus, deleteDetections)
-				)
+				new DefaultScriptableWorkflowStep("Select detections", "selectDetections()")
```

Comment:
I'm uneasy about the complexity, when the actual code shouldn't be operating through selecting/deselecting.
Why not simplify to this?
```groovy
convertDetectionsToPoints()
// If required...
removeDetections()
```